response_operations_ui/controllers/case_controller.py

```diff
@@ -193,9 +193,12 @@ def get_iac_count_for_case(case_id):
     return iac_count
 
 
-def get_case_events_by_case_id(case_id):
+def get_case_events_by_case_id_and_optional_category(case_id, categories=None):
```

Comment:
I'd probably leave the name of this as `get_case_events_by_case_id`, don't think we need to be so specific on what it's doing